snippets/base/templates/base/includes/snippet_as.js

```diff
@@ -421,7 +421,9 @@ function sendMetric(metric, callback, href) {
       }
       return;
     {% else %}
-      if (!SNIPPET_METRICS_URL || (Math.random() > SNIPPET_METRICS_SAMPLE_RATE) || (!metric)) {
+
+    if (gSnippetsMap.get('telemetryEnabled') !== true || !SNIPPET_METRICS_URL ||
```

Comment:
This should actually be `gSnippetsMap.get("appData.telemetryEnabled")` https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/snippets.md#expected-values-in-gsnippetsmap